src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const ManOrWomen =
+    sex === 'm' ? `${partnerName} is my wife` : `${partnerName} is my husband`;
```

Comment:
Better to change the naming for this variable to something more understandable.
Example: ``ManOrWomen``  - ``partnerRole``